doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc

```diff
@@ -75,22 +75,22 @@ image::cases/showcase-start-case.png[]
 +
 image::cases/case-management-invalid-spec.png[]
 +
-Click *Complete*. 
+Click *Complete*.
 +
-Instead of the *Task Inbox* showing the `Prepare hardware spec` as `Complete`, it shows as `Ready`. 
-. In Showcase, click *Refresh* in the upper-right corner. Notice that a `Prepare hardware task` appears in the *Completed* column and another appears in the *In Progress* column. 
+Instead of the *Task Inbox* showing the `Prepare hardware spec` as `Complete`, it shows as `Ready`.
+. In Showcase, click *Refresh* in the upper-right corner. Notice that a `Prepare hardware task` appears in the *Completed* column and another appears in the *In Progress* column.
 +
 image::cases/case-management-new-spec-task.png[]
 +
 This is because the first `Prepare hardware spec` task has been completed with the specification file `invalid.pdf`. As a result, the business rule causes the task and file to be discarded, and a new user task created.
-. In the {CENTRAL} *Task Inbox*, repeat steps 12 and 13, but upload the `valid-spec.pdf` file instead of `invalid.pdf`. 
-. Log out of {CENTRAL} and log back in again as `Katy`. 
-. Go to *Menu* -> *Track* -> *Task Inbox*. There are two `Manager approval` tasks for Katy, one with the `invalid.pdf` hardware specification file, the other with the `valid-spec.pdf` file. 
+. In the {CENTRAL} *Task Inbox*, repeat steps 12 and 13, but upload the `valid-spec.pdf` file instead of `invalid.pdf`.
+. Log out of {CENTRAL} and log back in again as `Katy`.
+. Go to *Menu* -> *Track* -> *Task Inbox*. There are two `Manager approval` tasks for Katy, one with the `invalid.pdf` hardware specification file, the other with the `valid-spec.pdf` file.
 . Open, claim, and complete each task:
 .. Check the `approve` box for the task that includes the `valid-spec.pdf` file, then click *Complete*.
 .. Do not check the `approve` box on the task with the `invalid.pdf` file, then click *Complete*.
-. Go to *Menu* -> *Manage* -> *Process Instances* and open the *Order for IT hardware* process instance. 
-. Open the *Diagram* tab. Note that the process both `Order rejected` and `Place order` tasks as completed. 
+. Go to *Menu* -> *Manage* -> *Process Instances* and open the *Order for IT hardware* process instance.
+. Open the *Diagram* tab. Note that the process both `Order rejected` and `Place order` tasks as completed.
```

Comment:
Note that.. I think you are missing some words in this last sentence. I don't get it.

Reply:
Fixed.